src/crypto/shelley/utils.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+// TODO
+// import {Transaction} from 'react-native-chain-libs'
+import {BigNumber} from 'bignumber.js'
+
+export function getTxInputTotal(
+  // tx: Transaction,
+  tx: any,
+): BigNumber {
+  let sum = new BigNumber(0)
+
+  const inputs = tx.inputs()
```

Comment:
this would look much cleaner with `tx.inputs().reduce(...)`

Reply:
you can't do this because `tx.inputs` is not iterable. It's a webassembly class with two methods: `size` and `get`.